### ISSUE TITLE
Listener Position

### ISSUE THREAD

**Issue by zya** [Ircam-RnD/binauralFIR#7]
Any considerations on how we can change the listener position in real time? 
Similar to the native `listener.setPosition` and `listener.setOrientation`.

**Comment by ouhouhsami** [comment]
Hi, 

You should just use setPosition(azimuth, elevation, distance). The module, and this method is made explicitly for that use case.

Best

**Comment by zya** [comment]
From what I understood, the binauralFIR's `.setPosition` method is more like `panner.setPosition`. 
My question was about the `listener`. In the web audio spec, the `panner` node will spatialise the object relative to the `listener` position. 

So for example, if my audio graph has 2 instances of `binauralFIR` nodes, each set with different parameters, calling `listener.setPosition` should influence the parameters on each of the instances of `binauralFIR`. 

Does the library currently support this?

**Comment by ouhouhsami** [comment]
You are right, the library currently doesn't support that. This should be a new feature, with an API corresponding to the Web Audio API listener position.
